metadata.json

```diff
@@ -0,0 +1,56 @@
+{
+  "author": "Vox Pupuli",
+  "license": "MIT",
+  "name": "puppet-anubis",
+  "version": "0.0.1-rc0",
+  "summary": "A Puppet module to install Anubis",
+  "source": "https://github.com/voxpupuli/puppet-anubis",
+  "dependencies": [
+    {
+      "name": "puppetlabs-stdlib",
+      "version_requirement": ">= 4.25.0 < 10.0.0"
+    }
+  ],
+  "requirements": [
+    {
+      "name": "puppet",
+      "version_requirement": ">= 7.0.0 < 9.0.0"
+    },
```

Comment:
At this point, just drop Puppet for new modules?

Reply:
Will Forge accept that? I'd prefer not to play guineapig just yet